src/kinetics/GasKinetics.cpp

```diff
@@ -61,17 +61,18 @@ void GasKinetics::update_rates_T()
 
 void GasKinetics::update_rates_C()
 {
-    thermo().getActivityConcentrations(m_conc.data());
+    thermo().getActivityConcentrations(m_act_conc.data());
+    thermo().getConcentrations(m_phys_conc.data());
     doublereal ctot = thermo().molarDensity();
 
     // 3-body reactions
     if (!concm_3b_values.empty()) {
-        m_3b_concm.update(m_conc, ctot, concm_3b_values.data());
+        m_3b_concm.update(m_phys_conc, ctot, concm_3b_values.data());
     }
 
     // Falloff reactions
     if (!concm_falloff_values.empty()) {
-        m_falloff_concm.update(m_conc, ctot, concm_falloff_values.data());
+        m_falloff_concm.update(m_phys_conc, ctot, concm_falloff_values.data());
```

Comment:
Yes, that was intentional on my part. I was interpreting the role of the third-body collider in `falloff` reactions to be analogous to its role in `three-body` reactions. I would appreciate @rwest or @cfgoldsmith weighing in one way or the other.

Reply:
In either case, the inconsistency between `m_conc` and `ctot` in the above expression exists for falloff reactions in the current implementation, the same as for `three-body` reactions.

Reply:
I agree that falloff and three-body should be treated the same way. The most detailed discussion I've found of these issues is in https://doi.org/10.1016/j.combustflame.2019.11.041 and I think my reading of it (or my intuition) would suggest we use the actual concentration for both, but I guess the important thing is to do the same as whatever the rate coefficient's "author" did when fitting it.  That article also throws some shade at "Most combustion codes" for not yet implementing recently proposed mixture rules. 
We could ask @The-Burke-Lab to comment.

Reply:
> the important thing is to do the same as whatever the rate coefficient's "author" did when fitting it.

Ah, a very good point.  Which is making this start to sound more like our discussions of thermo and the "reference" vs. "standard" vs. whatever state... 😓

My _guess_ would be that most reaction fitting is done using an ideal gas assumption, which would be convenient, since the actual physical concentration and activity concentration are the same.